### PR TITLE
[CHF-138] Z-Wave Aeon MultiSensors are health checked

### PR DESCRIPTION
**Pull request by jackchi**
Most Aeon sensors are configured to report back `5-8 minutes`. 
ZigBee devices are now set to `2hrs`.  I propose setting this to `1hr`. 

cc 
@mckeed @mortent

### CODE REVIEW

File: devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy [hunk 1]
```diff
@@ -12,14 +12,17 @@
...
 		capability "Motion Sensor"
 		capability "Temperature Measurement"
 		capability "Relative Humidity Measurement"
 		capability "Illuminance Measurement"
 		capability "Configuration"
 		capability "Sensor"
 		capability "Battery"
+		capability "Health Check"
+
+		command "configureAfterSecure"
```
Comment (jackchi) [inline devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy:25]: The `Aeon MultiSensor Gen5` valueTile `configureAfterSecure` is the command on the tile. Need to add this or else that configure tile doesn't actually call `configure`. 
@mckeed - is this the right approach?

**Comment by mckeed** [comment]
Looks good except for in the Multi 6 DTH, the report interval can be changed in the preferences, so the checkInterval should be based on that. The others should eventually have the same setting, but it can be changed when that's added.

File: devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy (hunk 1 above)
Comment (mckeed) [inline devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy:25]: That's fine. I'm of the opinion that configure tiles should be taken out of all z-wave devices (see DVCSMP-264), but for now this is good.

**Comment by jackchi** [comment]
I see on Aeon MultiSensor 6 we need to have `checkInterval` be variable. 
We currently only send `checkInterval` in configure() upon updated/install. 

Do we want to punt Aeon6 this for now? Or keep 1 hour?

**Comment by mckeed** [comment]
Not sure what you mean? Why can't you just set the value to something like `timeOptionValueMap[reportInterval] * 2`

configure() gets called whenever the setting is changed

**Comment by jackchi** [comment]
All `checkInterval` change to 2\* (periodic reporting interval). Ready to merge. 
@mckeed @mortent

**Comment by lbredeso** [comment]
We shouldn't be adding new categorization mechanisms to DeviceTypes. If we need a capability more specific than "healthCheck", we should create multiple capabilities and implement them where appropriate.

👎

**Pull request closed without merging**